api/nakadi-event-bus-api.yaml

```diff
@@ -881,6 +891,48 @@ paths:
           format: int32
           required: false
           default: 10000
+        - name: commit_mode
+          in: query
+          description: |
+            The commit mode that will be used for this stream. Possible values:
+
+            - `commit_at_will`: client is free to choose how often he wants to commit. Of course the commit_timeout
+            property should still be respected.
+            When a batch is committed that also automatically commits all previous batches that were sent in a stream
+            (if the commit_semantics parameter is `per_partition` - that will commit all previous batches only for
+            current partition).
+
+            - `commit_each`: this commit mode requires each batch to be committed, and it should be done in correct
+            order. Committing a cursor when the previous is not yet committed is not possible.
+
+            - `auto_commit`: everything that Nakadi sends to stream is considered to be committed. No actions are
+            required from client side to commit. Be aware that with this commit mode loosing of some events is very
+            probable. So you should know what you are doing if you use autocommit.
+
+            Currently only `commit_at_will` is supported.
```

Comment:
I think this information should come first. If the other options are not supported I would feel like I lost my time reading information that is of no use.